.github/workflows/ci-events-helsinki-staging.yml

```diff
@@ -0,0 +1,129 @@
+name: Events Helsinki Build & Staging
+on:
+  push:
+    branches:
+      - main
+
+# allow only one run at a time
+concurrency: ${{ github.workflow }}
+
+env:
+  CONTAINER_REGISTRY: ghcr.io
+  CONTAINER_REGISTRY_USER: ${{ secrets.GHCR_CONTAINER_REGISTRY_USER }}
+  CONTAINER_REGISTRY_PASSWORD: ${{ secrets.GHCR_TOKEN }}
+  CONTAINER_REGISTRY_REPO: ghcr.io/city-of-helsinki/${{ github.event.repository.name }}
+  KUBECONFIG_RAW: ${{ secrets.KUBECONFIG_RAW_STAGING }}
+  BUILD_ARTIFACT_FOLDER: 'build_artifacts'
+  SERVICE_ARTIFACT_FOLDER: 'service_artifacts'
+  BASE_DOMAIN: ${{ secrets.BASE_DOMAIN_STAGING }}
+  SERVICE_PORT: 3000
+  PROJECT: events-helsinki
+  K8S_REQUEST_CPU: 50m
+  K8S_REQUEST_RAM: 150Mi
+  K8S_LIMIT_CPU: 200m
+  K8S_LIMIT_RAM: 400Mi
+  # horizontal pod autoscaler
+  K8S_HPA_ENABLED: true
+  K8S_HPA_MAX_REPLICAS: 3
+  K8S_HPA_MIN_REPLICAS: 1
+  K8S_HPA_MAX_CPU_AVG: 80 # 80% from request
+  #  K8S_HPA_MAX_RAM_AVG: 80 # 80% from request
+
+  K8S_LIVENESS_PATH: /api/healthz
+  K8S_READINESS_PATH: /api/readiness
+  NEXT_PUBLIC_SENTRY_ENVIRONMENT: 'test'
+  NEXT_PUBLIC_SENTRY_DSN: ${{ secrets.GH_SENTRY_DSN }}
+  NEXT_PUBLIC_SENTRY_TRACE_SAMPLE_RATE: 0
+  NEXTJS_DISABLE_SENTRY: 1
+  SENTRY_AUTH_TOKEN: ${{ secrets.GH_SENTRY_AUTH_TOKEN }}
+  NEXT_PUBLIC_CMS_GRAPHQL_ENDPOINT: https://tapahtumat.hkih.stage.geniem.io/graphql
+  NEXT_PUBLIC_EVENTS_GRAPHQL_ENDPOINT: https://tapahtumat-proxy.test.kuva.hel.ninja/proxy/graphql
+  NEXT_PUBLIC_LINKEDEVENTS_EVENT_ENDPOINT: https://api.hel.fi/linkedevents/v1/event
+  NEXT_PUBLIC_MATOMO_URL_BASE: '//webanalytics.digiaiiris.com/js/'
+  NEXT_PUBLIC_MATOMO_SITE_ID: 939
+  NEXT_PUBLIC_MATOMO_SRC_URL: 'piwik.min.js'
+  NEXT_PUBLIC_MATOMO_TRACKER_URL: 'tracker.php'
+  NEXT_PUBLIC_MATOMO_ENABLED: 0
+  NEXT_PUBLIC_APP_ORIGIN: https://${{ secrets.ENVIRONMENT_URL_STAGING }}
```

Comment:
Every application needs own (secret) variables for example
secrets.ENVIRONMENT_URL_STAGING
->
secrets.HARRASTUKSET_ENVIRONMENT_URL_STAGING
secrets.TAPAHTUMAT_ENVIRONMENT_URL_STAGING

Reply:
Added those 2 new secrets and changed the code. The old and needless `ENVIRONMENT_URL_STAGING ` is still left there. That should be removed later.

Reply:
Used `ENVIRONMENT_URL_STAGING_TAPAHTUMAT` and `ENVIRONMENT_URL_STAGING_HARRASTUKSET` (appendix) instead, because some of the variables needs a specific name prefix.

Reply:
Set `tapahtumat-mono.test.kuva.hel.ninja` for Tapahtumat, so it won't overlap with current live version of Tapahtumat.